_sass/_features.scss

```diff
@@ -99,6 +99,31 @@ body.features-page {
 
 .feature {
     margin-bottom: $spacing-unit/2;
+
+    .feature-link {
+        margin-left: -$spacing-unit !important;
+    }
+}
+
+.feature, .feature-group {
+    .feature-anchor {
+        top: -($header-height + $header-border-top-height + $header-border-bottom-height + $spacing-unit/2);
+        display: block;
+        position: relative;
+    }
+
+    .feature-link {
+        display: none;
+    }
+
+    &:hover {
+        &> .feature-link {
```

Comment:
Also this:
![zrzut ekranu z 2016-09-15 20-26-02](https://cloud.githubusercontent.com/assets/3450050/18562092/a7005cac-7b82-11e6-98e2-654c5b43513a.png)